.ci/coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# this is in a separate script from `.travis.yml` so we can set this option.
+shopt -s extglob
+
+wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz &&
+tar xzf master.tar.gz &&
```

Comment:
just so i have context...

will this all run on every commit?  can we cache the install directory via travis so that we only need to build if the cache is gone?  If so, this should probably be guarded by ` if [ -x tmp/kvoc ] ... ` or something?